{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/css/wysiwyg/wysiwyg.css

```diff
@@ -11,6 +11,14 @@
 
 /* We are styling root elements so we need access to type */
 /* stylelint-disable selector-no-type */
+p {
+  margin-bottom: 1vr;
+}
+
+ul {
+  list-style: disc;
+}
+
```

Comment:
Whenever i see our designs they rarely have anything close to the browser default. It's only a default in Chrome as well tbf.